services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedTenantService.java

```diff
@@ -482,47 +412,15 @@ public void stop(final Promise<Void> stopPromise) {
                 })
                 .recover(error -> {
                     if (MongoDbDeviceRegistryUtils.isDuplicateKeyError(error)) {
-                        logDuplicationError(error.getMessage(), tenantId, span);
+                        log.debug(String.format("The tenant [%s] already exists or an existing tenant uses a certificate authority with the same Subject DN", tenantId));
+                        TracingHelper.logError(span, "The tenant with the given identifier already exists or an existing tenant uses a certificate authority with the same Subject DN");
```

Comment:
It is better to pass the error as another argument in the `log.debug(...)` and `TracingHelper.logError(...)`.

Reply:
done